docs/staking.md

```diff
@@ -0,0 +1,46 @@
+---
+id: staking
+title: Staking
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section provides a brief introduction into how staking works in the HydraDX network. If you are not experienced in staking in a Substrate-based network, we recommend that you read this before deciding to participate.
+
+The consensus mechanism used by HydraDX is called Nominated Proof-of-Stake (NPoS). NPoS is a variation of Proof-of-Stake and is used in Substrate-based blockchains such as Polkadot and Kusama. The two central actors in an NPoS environment are called [**validators**](#validators) and [**nominators**](#nominators). 
+
+### Validators
+
+Validators participate in the network by running validator nodes which provide the infrastructure allowing a Substrate-based network to operate securely. Validator nodes fulfill three functions which are of paramount importance to the consensus mechanism. In the first place, they validate the information contained in blocks, such as the identity of the parties and the subject of the contract. In the second place, validators participate in the production of new blocks based on the validity statements of other validators. In the third place, they guarantee the finality of blockchain transactions.
+
+An important characteristic of NPoS is that not all validators participate in the validating process at the same time. Only validators in the *active validator set* perform the above-mentioned operations and earn rewards for doing so. Generally speaking, the set of active validators is limited to a fixed amount of nodes, however in [HydraDX Snakenet](/snakenet) we will pursue a larger set in order to give more people the possibility to participate.
```

Comment:
did you clarify this?
my last information was targeting 300first, 900after. 
your text reads like we will have many slots now and a straight cut transitioning to mainnet. this will scare people off